cmd/akamai-purger/main.go

```diff
@@ -169,20 +168,26 @@ func (c *Config) useDeprecatedSettings() {
 	c.AkamaiPurger.Throughput.QueueEntriesPerBatch = DeprecatedQueueEntriesPerBatch
 }
 
+// CachePurgeClient is testing interface.
+type CachePurgeClient interface {
```

Comment:
```suggestion
type cachePurgeClient interface {
```

Doesn't need to be exported.